imports/plugins/core/accounts/client/containers/accountsDashboardContainer.js

```diff
@@ -33,7 +33,7 @@ const handlers = {
           Alerts.toast(i18next.t("admin.groups.addUserError", { err: err.message }), "error");
         }
         if (!err) {
-          Alerts.toast(i18next.t("admin.groups.addUserSuccess"), "success");
```

Comment:
So we don't directly display text here. Those message are just keys that are used to find translations. So the thing to do would be to try and find if this translation is missing from a [here](https://github.com/reactioncommerce/reaction-demo/blob/0e1421e6758feebc7366ca7a0034c74de9b4f909/private/data/i18n/en.json#L29) and add that key and a default English version

Reply:
Thanks @zenweasel , I will fix this.

Reply:
@zenweasel , I added the key and it works.  The code passed the liniting, but not the test, there are many failures !. What do you suggest me to do?

Reply:
@nadaa You still have the wrong approach. We can't hard-code an English-only message like that. You need to put back the key `admin.groups.addUserSuccess` and make it work using that. Otherwise you will break it for every language except English.

You also need to run the linter (`npm run lint`) and fix any linting errors (you can ignore warnings). That's what is causing the tests to fail.